libevmasm/Assembly.cpp

```diff
@@ -205,7 +205,8 @@ ostream& Assembly::streamAsm(ostream& _out, string const& _prefix, StringMap con
 	{
 		_out << _prefix << "stop" << endl;
 		for (auto const& i: m_data)
-			assertThrow(u256(i.first) < m_subs.size(), AssemblyException, "Data not yet implemented.");
```

Comment:
What is the situation when `m_data` contains a pair whose first element is small?  If that kind of pairs should be ignored, I guess that kind of pairs should not be inserted in the first place.  Is there a possibility that we are hiding problems here?